src/Jwt/LcobucciFactory.php

```diff
@@ -37,8 +37,12 @@ final class LcobucciFactory implements TokenFactoryInterface
     ];
 
     private $configurations;
+    private $jwtLifetime;
 
-    public function __construct(string $secret, string $algorithm = 'hmac.sha256')
+    /**
+     * @param int|null $jwtLifetime If not null, an "exp" claim is always set to now + $jwtLifetime (in seconds)
+     */
+    public function __construct(string $secret, string $algorithm = 'hmac.sha256', ?int $jwtLifetime = 3600)
```

Comment:
why hardcode 3600 instead of reading `ini_get('session.cookie_lifetime')`?

Reply:
Good idea. Fixed. I just added a fallback to 3600 in case the value is set to "session", as this concept doesn't exist for JWT.